scripts/bootstrap/buildenv.sh

```diff
@@ -301,3 +301,15 @@ function get_bind_target() {
   $BAZEL --bazelrc=${BAZELRC} --nomaster_bazelrc ${BAZEL_DIR_STARTUP_OPTIONS} \
     query "deps($1, 1) - $1"
 }
+
+# Create a link for a directory on the filesystem
+function link_dir() {
+  local source=$1
+  local dest=$2
+
+  if [ "${PLATFORM}" == "windows" ]; then
```

Comment:
nit: use [[ instead of [ because the former is a Bash built-in and the latter is a binary, so [[ is faster on Windows

Reply:
I copied that from line https://source.bazel.build/bazel/+/master:scripts/bootstrap/buildenv.sh;l=104?q=buildenv.sh, should I update that as well, or leave it for a later patch?

Reply:
leave it for a later patch; let's just not add new code that uses [